internal/tasks/manifests.go

```diff
@@ -532,12 +541,32 @@ func (j *Janitor) CheckVulnerabilitiesForNextManifest() (returnErr error) {
 		}
 	}()
 
-	//find manifest to sync
+	return j.checkVulnerabilitiesForManifest(vulnCheckSelectQuery)
+}
+
+// CheckVulnerabilitiesForNextPendingManifest is very similar to CheckVulnerabilitiesForNextManifest
+// with the exception that it only checks for already to clair submitted manifests which are still
+// in vuln_status Pending
```

Comment:
It also checks manifests that were never submitted to Clair (when a manifest is first added, it's vuln_status is initialized as "Pending").